ansible/ocs.yaml

```diff
@@ -0,0 +1,292 @@
+      block:
+        - name: Set msdos disk extended partition range end
+          set_fact:
+            extended_part_end: "100%"
+          when: device_info.disk.table == "msdos"
+
+        - name: Set non-msdos disk extended partition range end
+          set_fact:
+            extended_part_end: "40%"
+          when: device_info.disk.table != "msdos"
+
+        - name: Create partitions on {{ ocs_device }}
+          shell: |
+            parted {{ ocs_device }} mkpart primary ext2 0% 10%;
+            parted {{ ocs_device }} mkpart primary ext2 10% 20%;
+            parted {{ ocs_device }} mkpart primary ext2 20% 30%;
+            parted {{ ocs_device }} mkpart extended 30% {{ extended_part_end }};
+            parted {{ ocs_device }} mkpart logical ext2 40% 50%;
+            parted {{ ocs_device }} mkpart logical ext2 50% 60%;
+            parted {{ ocs_device }} mkpart logical ext2 60% 70%;
+            parted {{ ocs_device }} mkpart logical ext2 70% 80%;
+            parted {{ ocs_device }} mkpart logical ext2 80% 90%;
+            parted {{ ocs_device }} mkpart logical ext2 90% 100%
+
+    - name: Re-read host {{ ocs_device }} information
+      parted: device={{ ocs_device }} unit=GiB
+      register: device_info
+
+    - debug:
+        var: device_info
+
+    - name: Attach disks to storage VMs
+      command: "{{ item }}"
+      register: attach_disks
+      failed_when: attach_disks.stderr != "" and "already exists" not in attach_disks.stderr
+      with_items:
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-0 --source {{ ocs_device }}1 --target vdb --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-0 --source {{ ocs_device }}2 --target vdc --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-0 --source {{ ocs_device }}3 --target vdd --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-1 --source {{ ocs_device }}5 --target vdb --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-1 --source {{ ocs_device }}6 --target vdc --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-1 --source {{ ocs_device }}7 --target vdd --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-2 --source {{ ocs_device }}8 --target vdb --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-2 --source {{ ocs_device }}9 --target vdc --persistent"
+        - "virsh attach-disk {{ ocp_cluster_name }}-storage-2 --source {{ ocs_device }}10 --target vdd --persistent"    
+
+    - name: Fixes for dev-scripts vbmc container
+      block:
+      - name: Stop and remove vbmc container
+        shell: podman stop vbmc && podman rm vbmc
+
+      - name: Start vbmc container with proper mounts
+        shell: podman run -d --net host --privileged --name vbmc -v "{{ base_path }}/virtualbmc/vbmc":/root/.vbmc -v "/root/.ssh":/root/ssh -v "/var/run/libvirt:/var/run/libvirt:Z" quay.io/metal3-io/vbmc 
```

Comment:
Guessing this is required because of a dev-scripts bug? Can you describe it in a comment so we know when we can remove this.

Reply:
Will do.